### PR TITLE
feat(AdvancedView): introduce feature flag for discord import tool

### LINKED ISSUE #6843: Import-tool: add feature flag to desktop to enable/disable discord import support
This is so that https://github.com/status-im/status-desktop/pull/6441 doesn't grow too big and has to be rebased forever until it can land.
The feature flag will enables us to land individual pieces of the discord tool without breaking or interfering with existing functionality.

### PR DESCRIPTION
**Pull request by PascalPrecht**
This adds a feature flag for the discord import tool so we can start
landing individual pieces of the feature without it being fully
implemented.

It also introduces the modal chooser for creating new communities but
it doesn't do anything more than that, as of this commit

Co-authored with @caybro 

Closes #6843

### CODE REVIEW

**Comment by PascalPrecht** [comment]
Screenshot from 2022-08-18 18-29-37
Screenshot from 2022-08-18 18-29-52

File: ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml [hunk 1]
```diff
@@ -6,13 +6,16 @@ import StatusQ.Core 0.1
...
+import StatusQ.Popups 0.1
+import StatusQ.Popups.Dialog 0.1
 
 import utils 1.0
 import shared.popups 1.0
 
 import "controls"
 import "stores"
 import "popups"
+import "../../AppLayouts/Chat/panels/communities" // TODO correct import? or move somewhere else?
```
Comment (osmaczko) [inline ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml:18]: Incorrect :P We must not use components from different AppLayouts. Please extract the needed component to the shared section.

File: ui/app/AppLayouts/Profile/views/AdvancedView.qml [hunk 2]
```diff
@@ -51,8 +53,7 @@ SettingsContentBase {
                 }
             }
 
-            // TODO: replace with StatusQ component
-            StyledText {
+            StatusBaseText {
```
Comment (osmaczko) [inline ui/app/AppLayouts/Profile/views/AdvancedView.qml:56]: :+1:

**Review by osmaczko — CHANGES_REQUESTED** [review]
LGTM, besides one important qml architecture violation

**Review by noeliaSD — APPROVED** [review]
LGTM!

File: ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml (hunk 1 above)
Comment (PascalPrecht) [reply ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml:19]: Thanks, this now lives in `ui/imports/shared/panels` (see first commit of this PR)

Comment (osmaczko) [reply ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml:19]: Is this import still needed then? I guess you forgot to remove it :sweat_smile:

**Review by osmaczko — APPROVED** [review]

**Pull request merged**